README.md

```diff
@@ -339,49 +337,49 @@ Surveyor Jobs discover samples to download/process along with
 recording metadata about the samples. A Surveyor Job should queue
 `Downloader Jobs` to download the data it discovers.
 
-The Surveyor can be run with the `./foreman/run_surveyor.sh`
+The Surveyor can be run with the `./foreman/run_management_command.sh`
 script. The first argument to this script is the type of Surveyor Job
 to run, which will always be `survey_all`.
 
 Details on these expected arguments can be viewed by
 running:
 
 ```bash
-./foreman/run_surveyor.sh survey_all -h
+./foreman/run_management_command.sh survey_all -h
 ```
 
 The Surveyor can accept a single accession code from any of the source
 data repositories (e.g., Sequencing Read Archive,
  ArrayExpress, Gene Expression Omnibus):
 
 ```bash
-./foreman/run_surveyor.sh survey_all --accession <ACCESSION_CODE>
+./foreman/run_management_command.sh survey_all --accession <ACCESSION_CODE>
```

Comment:
What happens when you survey something locally now that we ripped out Nomad? Might be good to touch on this briefly somewhere in the docs

Reply:
This is a good question but I don't actually have a good answer. I just made https://github.com/AlexsLemonade/refinebio/issues/2775

Do you think I should address that before merging to dev?

Reply:
I don't think it's worth blocking everything else, so we can probably put it off until next sprint